ui/app/adt/views/wardList.html

```diff
@@ -3,14 +3,15 @@
     <span ng-if="tableDetails.length == 0" class="no-data-info">
         {{ 'EMPTY_WARD_MESSAGE'|translate }}
     </span>
-    <table class="table ward-details-content" ng-if="tableDetails.length > 0">
-        <thead>
+    <div style="overflow-x: scroll">
```

Comment:
@Arjun-Go @binduak I don't see any impact that this change brings in, if this change was intentional, could you please explain the rationale behind it?

Reply:
After checking with @Phanindra-tw, I got to know that this is needed for smaller screens, because the search box was scrolling along when horizontal scroll was done.